app/views/layouts/application.html.erb

```diff
@@ -26,20 +26,22 @@
     })(window,document,'script','dataLayer','GTM-N58Z5PG');</script>
     <!-- End Google Tag Manager -->
 
+    <% gtm_id = Rails.env.production? ? 'G-5707QPHZE0' : 'G-LR5YHTJD65' %>
```

Comment:
Whoa, setting local variables in views feels... dirty somehow.

I think it can stay though.

Reply:
i know! but it would also feel weird to me to define this specific variable in the controller